config/routes.rb

```diff
@@ -11,7 +11,7 @@
           delete 'signout', to: 'sessions#destroy'
         end
       end
-      resources :users, only: [:create]
+      resources :users, only: %i[create update destroy]
```

Comment:
Why `destroy`?

Reply:
It shouldn't be there.. I am working on the Delete User feature in parallel.